app/screens/more_channels/more_channels.js

```diff
@@ -362,8 +396,24 @@ export default class MoreChannels extends PureComponent {
                             value={term}
                         />
                     </View>
+                    <Picker
+                        selectedValue={this.state.typeOfChannels}
+                        style={{height: 50, width: 250}}
+                        onValueChange={(itemValue) =>
+                            this.setState({typeOfChannels: itemValue})
+                        }
+                    >
+                        <Picker.Item
+                            label='Show: Public Channels'
+                            value='public'
+                        />
+                        <Picker.Item
+                            label='Show: Archived Channels'
```

Comment:
translatable as the above.

it should be something like `label={intl.formatMessage({id: 'mobile.more_channels.archived', defaultMessage: 'Archived Channels'})}`

once you do this make sure to run `make i18n-extract`